maintainers/maintainer-list.nix

```diff
@@ -2312,7 +2312,11 @@
     email = "tierpluspluslists@gmail.com";
     name = "Karn Kallio";
   };
-
+  klntsky = {
```

Comment:
You can use your github name here.
```nix
{
  "8084" = {
    # ...
  };
}
```

```nix
{
  maintainers = [ maintainers."8084" ];
}
```

Reply:
Thanks, I'd rather prefer to leave it as it is.